### PR TITLE
[docs] Update compaction metrics in 282 

### PR DESCRIPTION
**Pull request by sijia-w**
This is a follow-up of #11739.

### CODE REVIEW

**Comment by sijia-w** [comment]
@Technoboy- @Anonymitaet  Please help review the changes and leave a comment if anything should be modified. Thank you.

**Review by Technoboy- — APPROVED** [review]
LGTM

File: site2/docs/reference-metrics.md [hunk 1]
```diff
@@ -200,20 +200,19 @@ All the topic metrics are labelled with the following labels:
...
-| pulsar_in_messages_total | Counter | The total number of messages received for this topic |
-| pulsar_out_bytes_total | Counter | The total number of bytes read from this topic |
-| pulsar_out_messages_total | Counter | The total number of messages read from this topic |
-| pulsar_compaction_removed_event_count | Gauge | The removed event count of compaction |
-| pulsar_compaction_succeed_count | Gauge | The succeed count of compaction |
-| pulsar_compaction_failed_count | Gauge | The failed count of compaction |
-| pulsar_compaction_duration_time_in_mills | Gauge | The duration time of compaction |
-| pulsar_compaction_read_throughput | Gauge | The read throughput of compaction |
-| pulsar_compaction_write_throughput | Gauge | The write throughput of compaction |
+| pulsar_in_bytes_total | Counter | The total number of messages in bytes received for this topic. |
```
Comment (Anonymitaet) [inline site2/docs/reference-metrics.md:203]:
```suggestion
| pulsar_in_bytes_total | Counter | The total number of messages received for this topic (in bytes). |
```

if agree, please update all, thanks

**Review by Anonymitaet — COMMENTED** [review]
This PR modifies docs for `master`. Will you modify docs for 2.8.2?

**Comment by sijia-w** [comment]
@Anonymitaet I prefer to not use the parenthesis, that way, "in bytes" looks like a note. The main info in this sentence is still "the total number of messages".

**Comment by sijia-w** [comment]
@Anonymitaet I modified the doc in 2.8.2, and then I updated the changes to master. So this pr covers changes to both versions.

**Review by Anonymitaet — APPROVED** [review]

**Pull request merged**